software/anubis.yml

```
@@ -0,0 +1,12 @@
+name: "Anubis"
+website_url: "https://anubis.techaro.lol/"
+source_code_url: "https://github.com/TecharoHQ/anubis"
+description: "Web AI Firewall Utility which protects upstream resources from scraper bots."
```

I dont think those need to be capitalised.
```suggestion
description: "Web AI firewall utility which protects upstream resources from scraper bots."
```